src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   `OTEL_DOTNET_EXPERIMENTAL_METRICS_EMIT_OVERFLOW_ATTRIBUTE` to `true` before
   setting up the `MeterProvider`.
   ([#4737](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4737))
+* Remove Jaeger Exporter from the core SDK. Jaeger supports OTLP protocol natively.
```

Comment:
Jaeger was never part of core sdk..

Reply:
I guess we don't need changelog updates. JaegerExporter's changelog will be removed as part of the exporter removal.

Reply:
Removed the CHANGELOG entry.